scripts/make_luals_meta.py

```diff
@@ -42,6 +45,7 @@
 c2 = {}
 
 ---@class IWeakResource
+IWeakResource = {}
```

Comment:
Should this really define a table? Chatterino itself does not expose that table. `IWeakResource` is an interface some objects can implement.

Reply:
Since you can't access `IWeakResource:is_valid` in Lua code, I removed it. `Channel` already has `Channel:is_valid` defined (with the same doc comment).